src/button/styles.scss

```diff
@@ -110,40 +115,38 @@
   &.variant-flashbar-icon {
     // Icon has vertical padding, but no horizontal, we need to conpensate this
     // in order to have equal height and width
-    padding-left: awsui.$space-xxs;
-    padding-right: awsui.$space-xxs;
+    padding-inline: awsui.$space-xxs;
   }
 
   &.variant-modal-dismiss {
-    padding: awsui.$space-button-modal-dismiss-vertical awsui.$space-xxs;
-    margin-right: calc(-1 * #{awsui.$space-xxs});
+    padding-block: awsui.$space-button-modal-dismiss-vertical;
+    padding-inline: awsui.$space-xxs;
+    margin-inline-end: calc(-1 * #{awsui.$space-xxs});
   }
 
   &.variant-inline-icon {
-    padding-top: 0;
-    padding-bottom: 0;
-    border: 0;
+    padding-block: 0;
+    border-block: 0;
+    border-inline: 0;
```

Comment:
Are these really equivalent? Shouldn't be be like this?

```css
padding-block-start: 0;
padding-block-end: 0;
border: 0; // why changing this, if it is zero all way round?
```

Reply:
Logical properties have new single axis two property shorthands (i.e. `padding-block`, `padding-inline`). Four property shorthands (i.e. `margin`, `padding`, `border`) are a risk because they require that the 1st/3rd and 2nd/4th values are exactly the same, thus introducing the potential for bugs. Pull quote from the technical proposal:

> As seen above, the tradeoff with using logical properties is that some common physical shorthands (example: margin: 1px 2px 3px 4px) [do not have logical counterparts](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values/Margins_borders_padding#margin_shorthands). However, logical properties do offer new shorthands such as margin-inline and margin-block that were previously missing from the web platform. Additionally, there is an [ongoing proposal to add a flow relative syntax for shorthands](https://github.com/w3c/csswg-drafts/issues/1282). Therefore, the recommendation is to add physical shorthands to the disallowed list given the viable alternatives.

Reply:
Makes sense, but...

This type of migration converts it from a dummy no-brainer "find and replace type" of code review to additional scrutiny on every PR, because the CSS is not equivalent.

So here is the choice

1. Dumb down the changes to fully equivalent `padding-top` -> `padding-block-start`
2. Break it down into even more PRs, one component at most for each one